code/z_adventurezones/mad_mars.dm

```diff
@@ -522,15 +522,9 @@
 	update()
 	return
 
-/obj/vehicle/marsrover/relaymove(mob/user as mob, dir)
-	if(rider)
-		if(istype(src.loc, /turf/space))
-			return
-		icon_state = "marsrover2"
-		walk(src, dir, 2)
-	else
-		for(var/mob/M in src.contents)
-			M.set_loc(src.loc)
+/obj/vehicle/marsrover/do_special_on_move(mob/user as mob, dir)
+	icon_state = "marsrover2"
+	return
```

Comment:
```suggestion
```

Reply:
Is there a reason to remove the return from the end of this specific instance of do_special_on_move but not the others?

Because, like, they all do this.

I won't object to it, but if it's a style thing, wouldn't it make more sense to have them all consistent?